.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "eth"]
 	path = eth
 	url = https://github.com/darkforest-eth/eth
+[submodule "circuits"]
+	path = circuits
+	url = https://github.com/darkforest-eth/circuits.git
+[submodule "packages"]
+	path = packages
+	url = https://github.com/darkforest-eth/packages.git
```

Comment:
I actually think you don't want these packages submoduled.

Reply:
Yeah they dont technically do anything here since you're using the packaged copies right?

But we dont have a darkforest-notlocal and people are always confused about where stuff is coming from and would like to have the code editor be able to find it...

Reply:
Not exactly. Since you are adding them into the workspace, they would be linked in the yarn install process. If you don't add them to the workspace, then the code editor won't know about them

Reply:
So I think they should be added to allow users to have a more holistic understanding of the game (and source published by the core team)

Can you say why you dont think they should be added?

Reply:
The package repository is a reference to the source code that generated the npm published packages. Consumers should be exclusively using the published packages, as the source code of these packages have too much baggage of how they are built and published from the monorepo.

The source code is **only** useful for someone at the tippy-top of advanced, and actively harms the experience here.

Reply:
Now that dfdao has launched a community round using darkforest-local and they are writing up docs on how to do it with the repo, this should become more approachable for folks to spin everything up. So we should merge the addition of packages/circuits submodules.